Cargo.toml

```diff
@@ -63,15 +68,16 @@ members = ["libs/test-utils"]
 exclude = ["scripts/attestation_agent/app"]
 
 [features]
-default = ["snapshot-overlayfs", "signature-cosign"]
+default = ["snapshot-overlayfs", "signature-cosign", "keywrap-grpc"]
 kata-cc = ["encryption", "keywrap-grpc", "snapshot-overlayfs", "signature-cosign"]
 kata-cc-s390x = ["encryption", "keywrap-grpc", "snapshot-overlayfs", "signature-simple"]
 enclave-cc = ["encryption", "keywrap-native", "eaa-kbc", "snapshot-unionfs", "signature-simple"]
 
 encryption = ["ocicrypt-rs/block-cipher"]
 keywrap-cmd = ["ocicrypt-rs/keywrap-keyprovider-cmd"]
-keywrap-grpc = ["ocicrypt-rs/keywrap-keyprovider-grpc", "prost", "tonic"]
+keywrap-grpc = ["ocicrypt-rs/keywrap-keyprovider-grpc", "prost", "tonic", "tonic-build"]
 keywrap-native = ["ocicrypt-rs/keywrap-keyprovider-native", "attestation_agent"]
+keywrap-ttrpc = ["dep:ttrpc", "dep:protobuf", "ttrpc-codegen"]
```

Comment:
`ocicryp-rs/keywrap-keyprovider-ttrpc` feature can add here

Reply:
Thanks! Nice suggestions! I'd force push after @HaokunX-intel had ensured the fixes.